crud/common/utils.lua

```diff
@@ -437,6 +441,45 @@ function utils.truncate_tuple_metadata(tuple_metadata, field_names)
     return truncated_metadata
 end
 
+function utils.cut_rows(rows, metadata, field_names, opts)
+    dev_checks('table', '?table', 'table', {
+        mapped = '?boolean',
+    })
+
+    opts = opts or {}
+
+    local truncated_metadata, err = truncate_tuple_metadata(metadata, field_names)
+
+    if err ~= nil then
+        return nil, err
+    end
+
+    if opts.mapped then
+        for i, row in ipairs(rows) do
+            rows[i], err = schema.filter_obj_fields(row, field_names)
+
+            if err ~= nil then
+                return nil, err
+            end
+
+        end
+
+        return {
+            metadata = truncated_metadata,
+            rows = rows,
+        }
```

Comment:
It's the same as below. You could remove this part and place  `chema.truncate_row_trailing_fields` under `else` branch.